packages/firestore/console.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import './src/platform_browser/browser_init';
+
+export {
+  Firestore,
+  FirestoreDatabase,
+  PublicCollectionReference as CollectionReference,
```

Comment:
Hrm.  Could we export CollectionReference from firestore-types instead (same for the below types)? It seems like we should only need to export the actual classes for things with constructors / static methods.  In general I'd prefer stick to exporting curated types rather than the raw implementations (as per my usual reasons), but since the console is internal I'm a little more willing to play fast-and-loose.